tasks/check-deployer-job.yaml

```diff
@@ -76,8 +76,24 @@
     anarchy_finish_action:
       state: failed
 
-# FIXME - How to handle successful deployment where we did not get the
-#         completion callback?
+- name: Handle tower job success when no callback received
+  vars:
+    __job_status: "{{ r_api_response.json.status }}"
+    __job_name: "{{ r_api_response.json.name | default('UNKNOWN') }}"
+  when: __job_status in ['successful']
+  block:
+
+    - name: Notification message
+      debug:
+        msg:
+          - "No callback received for successful job: {{ __job_name }}"
+
+    - name: Handle action complete for {{ anarchy_subject_name }}
+      include_tasks: >-
+        {%- if anarchy_action_config_name is defined -%}
+        handle-action-{{ anarchy_action_config_name }}-complete.yaml
+        {%- endif -%}
+      ignore_errors: true
```

Comment:
We will need to revisit this later... for the AgnosticD deployer we cannot safely proceed without receiving the callback. But we can address this later.

Reply:
@jkupferer, ah thanks, a lefotver from my testing. I've removed that now so it will fail on errors.

Regarding AgnosticD deployer, I will look to add a when condition to limit conditions for that file being included